jdi-light/src/test/java/io/github/epam/properties/ScreensFolderTests.java

```diff
@@ -0,0 +1,45 @@
+package io.github.epam.properties;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.security.InvalidParameterException;
+import java.util.Properties;
+
+import static com.epam.jdi.light.common.Property.SCREENS_FOLDER;
+import static com.epam.jdi.light.common.PropertyValidationUtils.validateProperties;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public class ScreensFolderTests {
+
+    @DataProvider
+    public static Object[] negativeData() {
+        return new Object[]{"cd:\\fd", "c:\\f[d", "C:\\Program*Files\\tests"};
+    }
+
+    @DataProvider
+    public static Object[] positiveData() {
+        return new Object[]{"C:\\ProgramFiles\\tests", "D:\\tests", "c:\\files"};
```

Comment:
Lets add some more checks including predefined path, relative path and unix path:
- my/predefined/path
- /my/relative/path
- C:\my\win\absolute\path

Reply:
Done